server/status_server.py

```
@@ -85,19 +77,13 @@ def do_GET(self):
-                            "Skipping stale status file %s (age %.1fs)",
-                            status_file,
-                            age if age is not None else float("nan"),
-                        )
                         continue
 
                     aggregate["active_instances"] += int(
                         data.get("active_instances", 0)
                     )
                 except Exception as exc:
-                    _LOGGER.debug("Failed to read %s: %s", status_file, exc)
+                    continue
```

Ruff (0.14.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)